src/shared-components/carousel/arrow/style.ts

```diff
@@ -19,4 +20,29 @@ const ArrowContainer = styled.div<{
   ${({ disabled }) => disabled && `display: none;`};
 `;
 
-export default ArrowContainer;
+export const BottomRightAlignedArrowContainer = styled.div<{
+  prev: boolean;
+  next: boolean;
+  disabled: boolean;
+}>`
+  display: block;
+  align-self: flex-end;
+  margin: ${SPACER.small};
+
+  ${({ prev }) => prev && `order: 2; margin-left: auto;`};
```

Comment:
Im not sure about this flex-wrap thing and it wouldn't work with dots. Could you do me a favor and try with 

```
position:absolute;
bottom: -32px;
right: 32px;
```

32 is just a random number to get you an idea and the prev button would need more than next. Please try this and let me know (also remove flex:wrap)

Reply:
I tried this and it doesn't work because of the height of the Carousel component. Half of the arrows end up being hidden. It would only work if I add `overflow: visible` to `InnerContainer` or increase the `padding-bottom` of the `InnerContainer` or `slick-slider`, In my opinion, these seem like worse alternatives to `flex: wrap`. To remedy the dot problem, I could just force dots to be hidden when using this new prop?

Reply:
ok, nvm then. you can also add that comment to the docs explainng this wont work with dots

Reply:
Yup. I added it in [here](https://github.com/curology/radiance-ui/pull/336/commits/f796aba9a110453b6ad5930c34b1aadc409ef29e)!